tools/nl/embeddings/README.md

```diff
@@ -65,6 +65,14 @@ variables.
    ./run.sh -c undata data/curated_input/undata/sheets_svs.csv data/alternatives/undata/*.csv
    ```
 
+   To generate the `bio_ft` embeddings:
+
+   ```bash
+   ./run.sh -c bio data/curated_input/bio/sheets_svs.csv data/alternatives/main/*.csv
+   ```
+
+   Note: Bio embeddings uses the alternatives from main dc for now.
```

Comment:
why would use main dc alternative for bio?

Reply:
bio still wants to be able to handle all the same queries for main dc as well & for POC I didn't have any special bio alternatives, so I just used the main dc alternatives .. I could make a new folder for bio alternatives and copy the files from main over to it if you think that's cleaner

Reply:
It's just the sv_descriptions.csv and duplicate_names.csv are not bio related and are duplicated from other preindex/<sub_folder>. What would be a better way to not duplicate but share them?